internal/exec/exec.go

```diff
@@ -0,0 +1,28 @@
+package exec
```

Comment:
Leading license comments is missing here (and also in `exec_test.go`), you can copy it over from another `.go` file.